hub/powertoys/run.md

```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+| Windows services                                                                     | yes                | `!`                       | `! alg` to search for the 'Application Layer Gateway' service to be started or stopped.                                                                                                                                                                                                              |
+| Shell command                                                                        | yes                | `>`                       | `> ping localhost` to do a ping query.                                                                                                                                                                                                                                                               |
+| Time and date                                                                        | yes                | `(`                       | `( time and date` shows the current time and date in different formats. `( calendar week::04/01/2022` shows the calendar week for the date '04/01/2022'.                                                                                                                                             |
+| Time zones                                                                           | yes                | `&`                       | `& Newfoundland` shows the current time in the time zone of Newfoundland.                                                                                                                                                                                                                            |
+| Unit converter                                                                       | yes                | `%%`                      | `%% 10 ft in m` to calculate the number of meters in 10 feet.                                                                                                                                                                                                                                        |
+| URIs                                                                                 | yes                | `//`                      | `//` to launch your default browser. `// docs.microsoft.com` to have your default browser go to   https://docs.microsoft.com. `mailto:` and `ms-settings:` links are supported.                                                                                                                      |
```

Comment:
please keep the line breakes for better view. that makes it easier to see the relevant information in the text